### PR TITLE
fixing querier deployment kube manifest example 404 error

### PR DESCRIPTION
**Pull request by rjshrjndrn**

### CODE REVIEW

File: docs/quick-tutorial.md [hunk 1]
```diff
@@ -163,7 +163,8 @@ thanos query \
 
 Go to the configured HTTP address, and you should now be able to query across all Prometheus instances and receive de-duplicated data.
 
-* _[Example Kubernetes manifest](https://github.com/thanos-io/kube-thanos/blob/master/manifests/thanos-querier-deployment.yaml)_
+* _[Example Kubernetes manifest](https://github.com/thanos-io/kube-thanos/blob/master/manifests/thanos-query-deployment.yaml
+)_
```
Comment (squat) [inline docs/quick-tutorial.md]: Could you please put this back on the same line?

**Comment by squat** [comment]
cc @kakkoyun yes, you're right, it definitely should, in fact running it locally uncovers some errors. I'm preparing a PR to fix this now :muscle:

File: docs/quick-tutorial.md (hunk 1 above)
Comment (rjshrjndrn) [reply docs/quick-tutorial.md]: Done

**Comment by squat** [comment]
thanks @rjshrjndrn :) plz sign the DCO and we can merge :taco:

**Comment by bwplotka** [comment]
> cc @kakkoyun yes, you're right, it definitely should, in fact running it locally uncovers some errors. I'm preparing a PR to fix this now muscle

Was it fixed? @squat

**Pull request merged**